scipy/stats/tests/test_stats.py

```diff
@@ -7971,3 +7971,63 @@ def test_rename_mode_method(fun, args):
     with pytest.raises(TypeError, match=re.escape(err)):
         fun(*args, method='exact', mode='exact')
 
+
+class TestExpectile:
+    def test_same_as_mean(self):
+        rng = np.random.default_rng(42)
+        x = rng.random(size=20)
+        assert_allclose(stats.expectile(x, alpha=0.5), np.mean(x))
+
+    def test_minimum(self):
+        rng = np.random.default_rng(42)
+        x = rng.random(size=20)
+        assert_allclose(stats.expectile(x, alpha=0), np.amin(x))
+
+    def test_maximum(self):
+        rng = np.random.default_rng(42)
+        x = rng.random(size=20)
+        assert_allclose(stats.expectile(x, alpha=1), np.amax(x))
+
+    @pytest.mark.parametrize("alpha", [0.2, 0.5, 0.8])
+    def test_expectile_properties(self, alpha):
+        """
+        See Section 6 of
+        I. Steinwart, C. Pasin, R.C. Williamson & S. Zhang (2014).
+        "Elicitation and Identification of Properties". COLT.
+        http://proceedings.mlr.press/v35/steinwart14.html
+        """
+        rng = np.random.default_rng(42)
+        n = 20
```

Comment:
I'm a bit concerned that this is too small to shake out issues (e.g. the x-vs.-y thing in the subadditivity did not fail). This should be a cheap test though, so the following shouldn't make such a difference.
```suggestion
        n = 2000
```
Doubly so if we're trying to pick up small differences around `alpha=0.5` (could be even larger from my POV).